src/main/resources/db/migration/v0.36.0.4__highthroughputexpressiondataset.sql

```diff
@@ -0,0 +1,103 @@
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+SELECT nextval('bulkload_seq'), 'HTP_EXPRESSION_DATASET_ANNOTATION', 'WB HTP Expression Dataset Annotation Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'HTP Expression Dataset Annotation Bulk Loads';
+
+INSERT INTO bulkscheduledload (id, cronschedule, scheduleactive)
+SELECT id, '0 0 22 ? * SUN-THU', false FROM bulkload WHERE backendbulkloadtype = 'HTP_EXPRESSION_DATASET_ANNOTATION';
+
+INSERT INTO bulkfmsload (id, fmsdatatype, fmsdatasubtype)
+SELECT id, 'HTP_EXPRESSION_DATASET_ANNOTATION', 'FB' FROM bulkload WHERE name = 'FB HTP Expression Dataset Annotation Load';
+INSERT INTO bulkfmsload (id, fmsdatatype, fmsdatasubtype)
+SELECT id, 'HTP_EXPRESSION_DATASET_ANNOTATION', 'MGI' FROM bulkload WHERE name = 'MGI HTP Expression Dataset Annotation Load';
+INSERT INTO bulkfmsload (id, fmsdatatype, fmsdatasubtype)
+SELECT id, 'HTP_EXPRESSION_DATASET_ANNOTATION', 'RGD' FROM bulkload WHERE name = 'RGD HTP Expression Dataset Annotation Load';
+INSERT INTO bulkfmsload (id, fmsdatatype, fmsdatasubtype)
+SELECT id, 'HTP_EXPRESSION_DATASET_ANNOTATION', 'SGD' FROM bulkload WHERE name = 'SGD HTP Expression Dataset Annotation Load';
+INSERT INTO bulkfmsload (id, fmsdatatype, fmsdatasubtype)
+SELECT id, 'HTP_EXPRESSION_DATASET_ANNOTATION', 'WB' FROM bulkload WHERE name = 'WB HTP Expression Dataset Annotation Load';
+INSERT INTO bulkfmsload (id, fmsdatatype, fmsdatasubtype)
+SELECT id, 'HTP_EXPRESSION_DATASET_ANNOTATION', 'ZFIN' FROM bulkload WHERE name = 'ZFIN HTP Expression Dataset Annotation Load';
+
+
+--Adding 3 extra vocabulary terms for HTPTags
+
+INSERT INTO vocabularyterm (id, name, vocabulary_id) SELECT nextval('vocabularyterm_seq'), 'epigenome', id FROM vocabulary WHERE vocabularylabel = 'data_set_category_tags';
+INSERT INTO vocabularyterm (id, name, vocabulary_id) SELECT nextval('vocabularyterm_seq'), 'functional genomics and proteomics', id FROM vocabulary WHERE vocabularylabel = 'data_set_category_tags';
+INSERT INTO vocabularyterm (id, name, vocabulary_id) SELECT nextval('vocabularyterm_seq'), 'karyotyping', id FROM vocabulary WHERE vocabularylabel = 'data_set_category_tags';
+
+-- Adding highthroughputexpressiondatasetannotation
+
+CREATE SEQUENCE highthroughputexpressiondatasetannotation_seq START WITH 1 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE TABLE highthroughputexpressiondatasetannotation (
+     id bigint NOT NULL,
+     datecreated timestamp(6) with time zone,
+     dateupdated timestamp(6) with time zone,
+     dbdatecreated timestamp(6) with time zone,
+     dbdateupdated timestamp(6) with time zone,
+     internal boolean DEFAULT false NOT NULL,
+     obsolete boolean DEFAULT false NOT NULL,
+     curie character varying(255),
+     modentityid character varying(255),
+     modinternalid character varying(255),
+     name character varying(255),
+     numberofchannels integer,
+     realtednote character varying(255),
```

Comment:
This should be `relatednote_id bigint` and have a FK to the note table (see comment above).  You will also need to create a 'HTP Expression Dataset Note Type' VocabularyTermSet that defines the one acceptable note type as being 'summary'.